xarray/static/css/style.css

```
@@ -128,28 +127,26 @@ body.vscode-dark {
   display: contents;
 }
 
-.xr-section-item input {
-  display: inline-block;
+.xr-section-item > input,
+.xr-group-box-contents > input {
+  display: block;
   opacity: 0;
   height: 0;
   margin: 0;
 }
 
-.xr-section-item input + label {
+.xr-section-item > input + label {
```

```suggestion
.xr-section-item > input + label,
.xr-var-item > input + label {
```

So the style is still applied to the "show attrs" and "show data repr" icons for inline variables.